lib/components/form/styled copy.js

```diff
@@ -0,0 +1,184 @@
+import styled, { css } from 'styled-components'
```

Comment:
Duplicate file??

Reply:
Probably it got in doing a bulk git add, it shouldn't be in here.